tests/search/components/TestSearchForm.js

```diff
@@ -7,24 +7,30 @@ const render = ReactTestUtils.renderIntoDocument;
 const findByTag = ReactTestUtils.findRenderedDOMComponentWithTag;
 
 describe('<SearchForm />', () => {
+  var onSearch;
+  var root;
+  var form;
+  var input;
+
+  beforeEach(() => {
+    onSearch = sinon.spy();
+    root = render(<SearchForm onSearch={onSearch} />);
+    form = findByTag(root, 'form');
```

Comment:
I suppose it does use the existing api without additional lookups is one reason. Although it is adding more to the JSX. So it's a bit of a six of one, 1/2 dozen of another type of thing.